### PR TITLE
add emotes for players

### PR DESCRIPTION
**Pull request by SohKey**

### CODE REVIEW

**Review by Sceat — COMMENTED** [review]
Is the `memes` subfolder necessary ?

**Comment by SohKey** [comment]
> Is the `memes` subfolder necessary ?

Yep, i will soon push a new version of the pack were there is multiple other folders in the texture one

**Comment by Sceat** [comment]
Please provide an explanation for the `memes` subfolder, and keep in mind this is the demo pack, aresrpg will only use https://github.com/aresrpg/resourcepacks-closed

**Comment by Sceat** [comment]
WHERE IS THE FUCKING DAMN EXPLANATION FOR THE `memes` SUBFOLDER BLYAT

**Comment by SohKey** [comment]
> 
lel the fucking damn explanation for the `memes` subfolder is on your DAMNN search engine juste type "what is a meme" :)

More seriously what there is to explain ? And where should i put it ?
image

**Comment by Sceat** [comment]
image

**Comment by Sceat** [comment]
image

I know what a meme is blyat, there is no usecase for this subfolder so if u have no explanations follow the flat structure and do `textures/emotes/` only

File: assets/minecraft/lang/en_us.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+  "language.region": "AresRPG Lands",
+  "sign.edit": "§eMontant de kAres: ",
```
Comment (DeltaEvo) [inline assets/minecraft/lang/en_us.json:3]: This seems unrelated to this PR

File: assets/minecraft/lang/en_us.json [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+  "aresrpg.emotes.ban": 1,
+  "aresrpg.emotes.sheesh": 2,
+  "aresrpg.emotes.pepe": 3,
```
Comment (DeltaEvo) [inline assets/minecraft/lang/en_us.json:6]:
```suggestion
  "aresrpg.emotes.ban": "1",
  "aresrpg.emotes.sheesh": "2",
  "aresrpg.emotes.pepe": "3",
```

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

**Review by DeltaEvo — APPROVED** [review]

**Pull request merged**